XCUITests/NavigationTest.swift

```diff
@@ -216,68 +216,59 @@ class NavigationTest: BaseTestCase {
         waitForValueContains(app.textFields["url"], value: website_2["moreLinkLongPressInfo"]!)
     }
 
-    /* Disabled due to issue 5581
     func testLongPressOnAddressBar() {
         //This test is for populated clipboard only so we need to make sure there's something in Pasteboard
         navigator.goto(URLBarOpen)
-        app.textFields["address"].typeText("www.google.com\n")
-        waitUntilPageLoad()
-        app.textFields["url"].press(forDuration:3)
-        app.tables.cells["menu-Copy-Link"].tap()
-        app.textFields["url"].tap()
-        // Since the textField value appears all selected first time is clicked
-        // this workaround is necessary
+        app.textFields["address"].typeText("www.google.com")
+        // Tapping two times when the text is not selected will reveal the menu
         app.textFields["address"].tap()
+        sleep(1)
```

Comment:
@isabelrios what do you suggest I use here other than "sleep()"?

Reply:
Maybe a waitForExistence for the next element to tap app.textFields["address"], but I'm  not sure if that's going to  work...have you tried removing the sleep? looks like iOS 13.4.1 is more stable test execution wise...so maybe we don't need it anymore...